.github/ISSUE_TEMPLATE/report-bug.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug Report
+about: Report erroneous or unexpected behavior.
+title: ""
+labels: bug
+assignees: ""
+---
+
+**Description**
+
+[Please add a clear description of the bug you are encountering, such as any details about your machine, the file where you think this bug may be, error messages, screenshots, versions of software used, etc.]
```

Comment:
Need to fix the line lengths. I would suggest a tool like Prettier. Feel free to give me a shout on Element if you need a quick rundown on how to use it :+1: